Dockerfile

```diff
@@ -0,0 +1,52 @@
+FROM ubuntu:xenial as base
+
+# Install system requirements
+RUN apt update && \
+    # Global requirements
+    DEBIAN_FRONTEND=noninteractive apt install -y language-pack-en git build-essential software-properties-common curl git-core libxml2-dev libxslt1-dev libmysqlclient-dev libxmlsec1-dev libfreetype6-dev libssl-dev swig gcc g++ \
```

Comment:
why are we shipping g++ and gcc?

Also, I'd love to see this in a standard location, like `requirements/edx/apt.pkglist` or something, but I can see this go either way.

Reply:
Hi, there is already a `requirements/system/ubuntu/apt-packages.txt` file. Maybe that can be used.

Reply:
I think g++ and gcc are needed to build various wheels, but I didn't confirm it.

Thank you morenol - I will check that out